templates/components/itilobject/fields/priority_matrix.html.twig

```diff
@@ -103,8 +103,8 @@ $(function() {
          url: "{{ path('/ajax/priority.php') }}",
          datatype: 'json',
          data: {
-            'urgency': $('#dropdown_urgency{{ rand }}').select2('data')[0].id ?? 0,
-            'impact':  $('#dropdown_impact{{ rand }}').select2('data')[0].id ?? 0,
+            'urgency': $('#dropdown_urgency{{ rand }}').select2('data') ? $('#dropdown_urgency{{ rand }}').select2('data')[0].id : 0,
```

Comment:
maybe this can be reduced to 
```suggestion
            'urgency': $('#dropdown_urgency{{ rand }}').select2('data')?.[0]?.id ?? 0,
```

Reply:
Didn't know that `?.` also worked to access array keys (`?.[0]`).
Works great 👍

Reply:
It will not work for Microsoft Edge on versions that were not build on a Chromium engine. This version was released in 2020, and maybe there are still lots of people using older versions.

Reply:
Ok then I revert